clients/hadoopfs/src/test/java/io/lakefs/auth/AWSLakeFSTokenProviderTest.java

```diff
@@ -17,14 +18,22 @@ public void testProviderIdentityTokenSerde() throws Exception {
         conf.set("fs.lakefs." + Constants.TOKEN_AWS_CREDENTIALS_PROVIDER_SESSION_TOKEN_KEY_SUFFIX, "sessionToken");
         conf.set("fs.lakefs." + Constants.TOKEN_AWS_STS_ENDPOINT, "https://sts.amazonaws.com");
 
-        AWSLakeFSTokenProvider provider = (AWSLakeFSTokenProvider)LakeFSTokenProviderFactory.newLakeFSTokenProvider(Constants.DEFAULT_SCHEME, conf);
+        AWSLakeFSTokenProvider provider = (AWSLakeFSTokenProvider) LakeFSTokenProviderFactory.newLakeFSTokenProvider(Constants.DEFAULT_SCHEME, conf);
```

Comment:
Not sure we need to cast here, it seems like an upcast that is usually implicit.

Reply:
i need it here because the `newLakeFSTokenProvider()` returns `LakeFSTokenProvider` 
while im testing specific code in `AWSLakeFSTokenProvider` (**AWS**) and i need the method `newPresignedGetCallerIdentityToken`